backend/public-graph/graph/resolver.go

```diff
@@ -2921,9 +2920,6 @@ func (r *Resolver) ProcessPayload(ctx context.Context, sessionSecureID string, e
 	defer updateSpan.Finish()
 
 	excluded, reason := r.isSessionExcluded(ctx, sessionObj, sessionHasErrors)
-	if excluded {
-		log.WithContext(ctx).WithFields(log.Fields{"session_id": sessionObj.ID, "project_id": sessionObj.ProjectID, "reason": *reason}).Infof("excluding session")
```

Comment:
I removed the log statements because they can be [quite noisy](https://github.com/highlight/highlight/pull/5123#discussion_r1181718382). Now that we have the reason recorded, we can refer to that for debugging if needed.